File: minting/src/main/scala/co/topl/minting/models/VrfHit.scala
package co.topl.minting.models

import co.topl.models.Slot
import co.topl.consensus.models.EligibilityCertificate
import co.topl.models.utility.Ratio

case class VrfHit(cert: EligibilityCertificate, slot: Slot, threshold: Ratio)

File: dart/minting/lib/models/vrf_hit.dart
import 'package:topl_protobuf/consensus/models/eligibility_certificate.pb.dart';

import 'package:fixnum/fixnum.dart';
import 'package:bifrost_common/models/ratio.dart';

class VrfHit {
  final EligibilityCertificate cert;
  final Int64 slot;
  final Ratio threshold;

  VrfHit(this.cert, this.slot, this.threshold);
}
